app/views/categories/_form.html.erb

```diff
@@ -49,7 +50,8 @@
       Users under this trust level cannot see that this category exists. Leave blank to leave the category public.
       This can be different (higher or lower) than the minimum posting trust level.
     </span>
-    <%= f.number_field :min_view_trust_level, class: 'form-element', min: 0, max: 10, step: 1 %>
+    <%= f.select :min_view_trust_level, options_for_select([['everyone', 0], ['anyone with an user account', 1], ['all but new users', 2], ['veteran users', 3], ['moderators only', 4], ['staff only', 5]], selected: @category.min_view_trust_level),
```

Comment:
"a user account" (not "an")

Reply:
TIL that there is used 'a' before 'user' and not 'an' (as I thought because of the 'u'; it appears to be, because it is spoken 'yu')